.travis.yml

```diff
@@ -20,114 +20,123 @@
 
 # This is the Travis CI top-level script to setup and run the NEST build and test environment. 
 
+language: python
 
-language: minimal
-
-# Required for Ubuntu Trusty (14.04 LTS).
+# Using the container infrastructure for Ubuntu Trusty (14.04 LTS).
 sudo: required
 dist: trusty
+
 #https://blog.travis-ci.com/2017-06-19-trusty-updates-2017-Q2
-group: edge
+group: deprecated-2017Q2
```

Comment:
From [that page](https://blog.travis-ci.com/2017-06-21-trusty-updates-2017-Q2-launch), this would only be valid until 9/1/17...  
Is this still related to the python versions problem that was discussed during the VC?
Otherwise can we not fix whatever goes wrong instead of working with the deprecated image?

Reply:
No more using the deprecated branch.